src/TodoList.jsx

```diff
@@ -18,16 +18,13 @@ const todoList = [
   ];
 
   
-function TodoList () {
-
+  function TodoList({ todoList }) {
```

Comment:
Now that you're passing `todoList` as a prop, delete the hard coded array above.